src/applications/search/containers/SearchApp.jsx

```diff
@@ -199,42 +199,47 @@ class SearchApp extends React.Component {
   };
 
   renderResults() {
-    const { loading, errors } = this.props.search;
+    const { loading, errors, currentPage, totalPages } = this.props.search;
     const hasErrors = !!(errors && errors.length > 0);
     const nonBlankUserInput =
       this.state.userInput &&
       this.state.userInput.replace(/\s/g, '').length > 0;
 
     // Reusable search input
     const searchInput = (
-      <form
-        onSubmit={this.handleSearch}
-        className="va-flex search-box"
-        data-e2e-id="search-form"
-      >
-        <input
-          type="text"
-          name="query"
-          aria-label="Enter the word or phrase you'd like to search for"
-          value={this.state.userInput}
-          onChange={this.handleInputChange}
-        />
-        <button type="submit" disabled={!nonBlankUserInput}>
-          <IconSearch color="#fff" />
-          <span>Search</span>
-        </button>
-      </form>
+      <div className="vads-u-background-color--gray-lightest vads-u-padding-x--2p5 vads-u-padding-bottom--1 vads-u-padding-top--1p5 vads-u-margin-y--1p5">
+        <div>Enter a keyword</div>
+        <form
+          onSubmit={this.handleSearch}
+          className="va-flex search-box vads-u-margin-top--1"
+          data-e2e-id="search-form"
+        >
+          <input
+            type="text"
+            name="query"
+            aria-label="Enter the word or phrase you'd like to search for"
+            value={this.state.userInput}
+            onChange={this.handleInputChange}
+          />
+          <button type="submit" disabled={!nonBlankUserInput}>
+            <IconSearch color="#fff" />
+            <span className="button-text">Search</span>
+          </button>
+        </form>
+      </div>
     );
 
     if (hasErrors && !loading) {
       return (
         <div className="usa-width-three-fourths medium-8 small-12 columns error">
-          <AlertBox
-            status="error"
-            headline="Your search didn't go through"
-            content="We’re sorry. Something went wrong on our end, and your search didn't go through. Please try again."
-            data-e2e-id="alert-box"
-          />
+          {/* this is the alert box for when searches fail due to server issues */}
+          <va-alert status="error" data-e2e-id="alert-box">
+            <h3 slot="headline">Your search didn't go through</h3>
+            <div>
+              We’re sorry. Something went wrong on our end, and your search
+              didn't go through. Please try again
+            </div>
+          </va-alert>
```

Comment:
Oh I have to do this. I might shamelessly peek at this haha.

Reply:
fun story, we were using the old alert box set up, and it just wasn't even connected to anything. checking to see if there was an error, but no code was written that would actually populate an error. This seemed as good a time as any to make the switch to actually doing some error handling!